examples/mcuboot/nsh/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_MCUBOOT_NSH
+	tristate "MCUboot NSH example"
```

Comment:
The "MCUboot NSH" name is not intuitive as it does not summarizes or describe the purpose of the example.
The example intends to demonstrate the swap upgrade process, this could be a good starting point for a new name.
Just a suggestion: "Swap upgrade example".